### PR TITLE
Improve rounding to month / weeks / days in Updates "Since Previous Update" column

### LINKED ISSUE #2839: Updates Dashboard Seems to Max Out at 1 Month
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

My updates dashboard does not seem to display "Since Previous Update" Lengths greater than one month. 

04/04/2022, 06:02:33 PM to 06/01/2022, 06:02:53 PM - Listed as 1 Month since the previous update.
the 

### Expected Behavior

04/04/2022, 06:02:33 PM to 06/01/2022, 06:02:53 PM - Listed as 8 weeks since the previous update.

### Steps To Reproduce

_No response_

### Relevant log output

```shell
teslamate_1  | 2022-08-06 19:11:00.757 [info] GET /
```

### Screenshots

Screen Shot 2022-08-06 at 3 03 02 PM

### Additional data

_No response_

### Type of installation

Docker

### Version

V1.27.1

### PR DESCRIPTION
**Pull request by swiffer**

### CODE REVIEW

File: grafana/dashboards/updates.json [hunk 1]
```diff
@@ -515,7 +511,7 @@
           }
         ]
       },
-      "pluginVersion": "11.1.0",
+      "pluginVersion": "11.2.0",
```
Comment (JakobLichterfeld) [inline grafana/dashboards/updates.json:514]: we are using 11.0.1 currently, aren't we?

**Review by JakobLichterfeld — CHANGES_REQUESTED** [review]
Awesome, ty.

File: grafana/dashboards/updates.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
       "type": "grafana",
       "id": "grafana",
       "name": "Grafana",
-      "version": "11.1.0"
+      "version": "11.2.0"
```
[the remark this answers is not in the document]
Comment (swiffer) [reply grafana/dashboards/updates.json:8]: True, I started testing Grafana 11.2.0 locally and did an export using Grafana 11.2.0. It shouldn't make a difference, if you like I can remove the changes not needed for now?

[the remark this answers is not in the document]
Comment (JakobLichterfeld) [reply grafana/dashboards/updates.json:8]: Yeah, would like to keep the PR only related to the issue and not mixing it up

**Comment by swiffer** [comment]
@JakobLichterfeld - done ;)

**Review by JakobLichterfeld — APPROVED** [review]
perfect, ty

**Pull request merged**